contracts/OwnerToken.sol

```diff
@@ -21,8 +21,8 @@ contract OwnerToken is BaseToken {
         BaseToken(_name, _symbol, 1, false, true, _baseTokenURI, address(this), address(this))
     {
         signerPublicKey = _signerPublicKey;
-        MasterToken masterToken = new MasterToken(_masterName, _masterSymbol, _masterBaseTokenURI, address(this));
-        emit MasterTokenCreated(address(masterToken));
+        MasterToken _masterToken = new MasterToken(_masterName, _masterSymbol, _masterBaseTokenURI, address(this));
+        emit MasterTokenCreated(address(_masterToken));
```

Comment:
This change isn't strictly necessary, but slither complained that `masterToken` would shadow `BaseToken.masterToken`, so I guess it wouldn't hurt to make slither happy here..